ExomeDepth/igv_xml_session.py

```diff
@@ -1,59 +1,333 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
+
 import argparse
 from string import Template
+from exomedepth_db import add_sample_to_db_and_return_refset_bam
 import settings
 
-def make_igvsession(igv_ed_umcu, igv_ed_hc, bam, vcf_hc, sample_id, vcf_SNV, axis, statistic):
-    template_file = Template(open(settings.template_xml).read())
-    new_session = "{0}_{1}_{2}_igv.xml".format(sample_id, statistic, args.runid)
-    igv_ed_hc_test = "{0}_{1}_test".format(igv_ed_hc, statistic)
-    igv_ed_umcu_test = "{0}_{1}_test".format(igv_ed_umcu, statistic)
-    bam_coverage = "{0}_coverage".format(bam)
-    bam_junctions = "{0}_junctions".format(bam)
-    min_axis, mid_axis, max_axis = axis
-    ratioid_UMCU = "{0}_UMCU".format(statistic)
-    ratioid_HC = "{0}_HC".format(statistic)
-    substitute_dic = {'session_var':new_session, 'igv_ed_umcu':igv_ed_umcu, 'igv_ed_hc':igv_ed_hc, 
-                    'bam':bam, 'vcf_hc':vcf_hc, 'sample_id':sample_id, 'igv_ed_hc_test':igv_ed_hc_test, 
-                    'igv_ed_umcu_test':igv_ed_umcu_test, 'bam_coverage':bam_coverage, 'bam_junctions':bam_junctions,  
-                    'vcf_SNV':vcf_SNV, 'min_axis':min_axis, 'mid_axis':mid_axis, 'max_axis':max_axis, 
-                    'ratioid_UMCU':ratioid_UMCU, 'ratioid_HC':ratioid_HC 
-                   }
+
+def parse_ped_family(ped_file, sampleid):
+    sample_dic = {}
+    for line in ped_file:
+        splitline = line.split()
```

Comment:
No `strip()` needed?

Reply:
In principle not, as last column is not parsed. But will be better, so inlcuded.